### ISSUE TITLE
CI failing for master branch

### ISSUE THREAD

**Issue by jabielecki** [helm/helm#7198]
Last few PRs off master all fail with:

```
$ go test ./cmd/...
"test-name" has been added to your repositories
search_repo.go:127: [debug] Original chart version: ""
search_repo.go:137: [debug] setting version to >0.0.0
search_repo.go:127: [debug] Original chart version: ""
search_repo.go:134: [debug] setting version to >0.0.0-0
search_repo.go:127: [debug] Original chart version: ""
search_repo.go:137: [debug] setting version to >0.0.0
search_repo.go:127: [debug] Original chart version: ">= 0.1, < 0.2"
search_repo.go:127: [debug] Original chart version: ">= 0.1, < 0.2"
search_repo.go:127: [debug] Original chart version: ">= 0.1"
search_repo.go:127: [debug] Original chart version: ">= 0.1"
search_repo.go:127: [debug] Original chart version: ""
search_repo.go:137: [debug] setting version to >0.0.0
search_repo.go:127: [debug] Original chart version: ""
search_repo.go:137: [debug] setting version to >0.0.0
search_repo.go:127: [debug] Original chart version: ""
search_repo.go:137: [debug] setting version to >0.0.0
search_repo.go:127: [debug] Original chart version: ""
search_repo.go:137: [debug] setting version to >0.0.0
search_repo.go:127: [debug] Original chart version: ""
search_repo.go:137: [debug] setting version to >0.0.0
--- FAIL: TestVersion (0.01s)
    --- FAIL: TestVersion/client_shorthand (0.00s)
        helm_test.go:59: running cmd:  version -c
        helm_test.go:65: does not match golden file testdata/output/version-client-shorthand.txt
            
            WANT:
            'version.BuildInfo{Version:"v3.0+unreleased", GitCommit:"", GitTreeState:"", GoVersion:""}
            '
            
            GOT:
            'version.BuildInfo{Version:"v3.0", GitCommit:"", GitTreeState:"", GoVersion:""}
            '
FAIL
FAIL	helm.sh/helm/v3/cmd/helm	4.662s
ok  	helm.sh/helm/v3/cmd/helm/require	0.026s
ok  	helm.sh/helm/v3/cmd/helm/search	(cached)
```

**Comment by bacongobbler** [comment]
will be fixed with #7199

**Comment by marckhouzam** [comment]
Closed by #7199
